subsys/bluetooth/host/gatt.c

```diff
@@ -5748,7 +5748,7 @@ void bt_gatt_encrypt_change(struct bt_conn *conn)
 
 	bt_gatt_foreach_attr(0x0001, 0xffff, update_ccc, &data);
 
-#if defined(CONFIG_BT_SETTINGS)
+#if defined(CONFIG_BT_SETTINGS) && defined(CONFIG_BT_GATT_SERVICE_CHANGED)
```

Comment:
Hi @jori-nordic
Seems that you (probably accidentally) cherry-picked the fix without comment alignment.

Reply:
Discussed offline. Will be fixed separately

Reply:
will do that in a subsequent PR to not block the releases.